Add Better Error Handling for API

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/seancallaway/mediamover/media.getPrimaryGenre({0xc000329148, 0x3}, {0xc0003291b0, 0x4})
        C:/Users/Sean/Projects/mediamover/media/tmdb.go:113 +0x72e
github.com/seancallaway/mediamover/media.ParseFilename({0xc000144060?, 0x0?}, 0x0)
        C:/Users/Sean/Projects/mediamover/media/main.go:29 +0x108
github.com/seancallaway/mediamover/cmd.init.func1(0xcb4b80?, {0xc00015e680, 0x1?, 0x2?})
        C:/Users/Sean/Projects/mediamover/cmd/movies.go:35 +0x165
github.com/spf13/cobra.(*Command).execute(0xcb4b80, {0xc00015e640, 0x2, 0x2})
        C:/Users/Sean/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0xaaa
github.com/spf13/cobra.(*Command).ExecuteC(0xcb4e60)
        C:/Users/Sean/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/Sean/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/seancallaway/mediamover/cmd.Execute()
        C:/Users/Sean/Projects/mediamover/cmd/root.go:21 +0x1a
main.main()
        C:/Users/Sean/Projects/mediamover/mediamover.go:8 +0xf
```

--- a/media/main.go
+++ b/media/main.go
@@ -26,6 +26,10 @@ func ParseFilename(filename string, tvShow bool) (Media, error) {
 		if err != nil {
 			return Media{}, err
 		}
-		return Media{Title: title, Year: year}, nil
+		genre, err := getPrimaryGenre(title, year)
+		if err != nil {
+			return Media{Title: title, Year: year}, err
+		}
+		return Media{Title: title, Year: year, Genre: genre}, nil
 	}
 }

--- a/media/tmdb.go
+++ b/media/tmdb.go
@@ -110,6 +110,10 @@ func getPrimaryGenre(title string, year string) (string, error) {
 		return "", errors.New("no movie found")
 	}
 
+	if len(movies.Results[0].GenreIds) == 0 {
+		return "", errors.New("no genre found")
+	}
+
 	genreId := movies.Results[0].GenreIds[0]
 
 	return genreMap[genreId], nil

